lxqt-build-tools version "0.8.0" required for compilation?

Apparently build instructions are a bit off.
They state:

The only runtime dependency is qtbase ≥ 5.7.1. In order to build CMake ≥ 3.0.2 and lxqt-build-tools >= 0.4.0 are needed as well as Git to pull translations and optionally latest VCS checkouts.

But I get:

```
mcon@cinderella:~/projects/qtermwidget/build$ cmake ..
-- The C compiler identification is GNU 9.3.0
-- The CXX compiler identification is GNU 9.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeLists.txt:39 (find_package):
  Could not find a configuration file for package "lxqt-build-tools" that is
  compatible with requested version "0.8.0".

  The following configuration files were considered but not accepted:

    /usr/share/cmake/lxqt-build-tools/lxqt-build-tools-config.cmake, version: 0.6.0



-- Configuring incomplete, errors occurred!
See also "/home/mcon/projects/qtermwidget/build/CMakeFiles/CMakeOutput.log".
```

I am running on a fairly up-to date Linux Mint 20.1 "Ulyssa".
My distribution already has binary distribution for qtermwidget, my only interest in compiling is I need to use this from Python (either PyQt5 or Pyside) and I was not able to find relevant bindings while I understand they should be available recompiling.

Build instructions for Python would be much appreciated.
I could help in preparing a setup.py suitable for general consumption (i.e.: for PyPi)

Thanks In Advance
Mauro

Update:
I have been able to overcome the above problem by manually lowering requirements in CMaleLists.txt, but I'm far from confident that is a good idea.

I also tried to build the PyQt5 bindings, but I got a bad error (apparently unrelated to the above):

```
mcon@cinderella:~/projects/qtermwidget/buildpy$ cmake ../pyqt/
CMake Warning (dev) in CMakeLists.txt:
  No project() command is present.  The top-level CMakeLists.txt file must
  contain a literal, direct call to the project() command.  Add a line of
  code such as

    project(ProjectName)

  near the top of the file, but after cmake_minimum_required().

  CMake is pretending there is a "project(Project)" command on the first
  line.
This warning is for project developers.  Use -Wno-dev to suppress it.

-- The C compiler identification is GNU 9.3.0
-- The CXX compiler identification is GNU 9.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PythonInterp: /usr/bin/python (found version "3.8.5") 
-- Found PythonLibs: /usr/lib/x86_64-linux-gnu/libpython3.8.so (found version "3.8.5") 
-- Python system site-packages directory: /usr/lib/python3/dist-packages
-- The Python files will be installed to /usr/local/lib/python3/dist-packages. Make sure to add them to the Python search path (e.g. by setting PYTHONPATH)
-- Found PythonLibrary: /usr/lib/x86_64-linux-gnu/libpython3.8.so  
-- Found SIP configuration but the sip executable could not be found.
Traceback (most recent call last):
  File "/home/mcon/projects/qtermwidget/pyqt/cmake/FindPyQt5.py", line 31, in <module>
    pyqt_sip_dir = next(
StopIteration

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mcon/projects/qtermwidget/pyqt/cmake/FindPyQt5.py", line 36, in <module>
    raise RuntimeError("Cannot find QtCore/QtCoremod.sip from candidates: " + repr(candidates))
RuntimeError: Cannot find QtCore/QtCoremod.sip from candidates: ['/usr/local/lib/python3.8/dist-packages/PyQt5/bindings', '/usr/share/sip/PyQt5']
-- Found PyQt5 version: 5.14.1
CMake Warning (dev) in CMakeLists.txt:
  No cmake_minimum_required command is present.  A line of code such as

    cmake_minimum_required(VERSION 3.16)

  should be added at the top of the file.  The version specified may be lower
  if you wish to support older CMake versions for this project.  For more
  information run "cmake --help-policy CMP0000".
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Configuring done
-- Generating done
-- Build files have been written to: /home/mcon/projects/qtermwidget/buildpy
mcon@cinderella:~/projects/qtermwidget/buildpy$ make
Scanning dependencies of target __home_mcon_projects_qtermwidget_buildpy____pycache_____init__.cpython-38.pyc
Byte-compiling /home/mcon/projects/qtermwidget/buildpy//__init__.py to /home/mcon/projects/qtermwidget/buildpy//__pycache__/__init__.cpython-38.pyc
[  0%] Built target __home_mcon_projects_qtermwidget_buildpy____pycache_____init__.cpython-38.pyc
CMakeFiles/python_module_QTermWidget.dir/build.make:71: *** missing separator.  Stop.
make[1]: *** [CMakeFiles/Makefile2:104: CMakeFiles/python_module_QTermWidget.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
mcon@cinderella:~/projects/qtermwidget/buildpy$ 
```
Any hint welcome.
TiA!

--- a/CHANGELOG
+++ b/CHANGELOG
@@ -1,3 +1,8 @@
+qtermwidget-0.17.0 / unreleased
+===============================
+  * Drop support for KDE3 color scheme formats. (#367)
+  * Added modes for background image. How background images are drawn is changed and manual reconfiguration is needed. See docs/configuration.md for more details. (#385)
+
 qtermwidget-0.16.1 / 2020-11-14
 ===============================
   * Bumped version to 0.16.1, for a point release of qterminal.

--- a/README.md
+++ b/README.md
@@ -52,15 +52,17 @@ License: public-domain
 
 ### Compiling sources
 
-The only runtime dependency is qtbase ≥ 5.7.1.
+The only runtime dependency is qtbase ≥ 5.12.0.
 Build dependencies are as follows:
-- CMake ≥ 3.0.2 serves as the build system and therefore needs to be present to compile.
--  [lxqt-build-tools](https://github.com/lxqt/lxqt-build-tools/) >= 0.4.0 is also needed for compilation.
--  Git is needed to pull translations and optionally pull latest VCS checkouts.
+- CMake ≥ 3.1.0 serves as the build system and therefore needs to be present to compile.
+- The latest [lxqt-build-tools](https://github.com/lxqt/lxqt-build-tools/) is also needed for compilation.
+- Git is needed to optionally pull latest VCS checkouts.
 
 Code configuration is handled by CMake. CMake variable `CMAKE_INSTALL_PREFIX` will normally have to be set to `/usr`, depending on the way library paths are dealt with on 64bit systems. Variables like `CMAKE_INSTALL_LIBDIR` may have to be set as well.
 
 To build, run `make`. To install, run `make install` which accepts variable `DESTDIR` as usual.
+
+To build PyQt bindings, specify an additional CMake option `QTERMWIDGET_BUILD_PYTHON_BINDING=ON` when building this library.
 
 ### Binary packages
 

--- a/docs/configuration.md
+++ b/docs/configuration.md
@@ -4,12 +4,17 @@ How the background image is drawn has been changed since version 0.17.
 Intuitively, the background image is now drawn above the background color instead of below it.
 Technically, the background image is no longer blended with the background color.
 
-Before migrating the settings, the background image should be converted to a translucent one with the transparency level matching the original terminal transparency.
-For example, if the original terminal transparency of qtermwidget was 25%, the converted image should have transparency 25%.
+Any background image can be used but, of course, it should be chosen so that the terminal text can be easily read on it.
+Since an image may not be totally dark or light, you might want to use a translucent image as the background.
+As a result, the background image is mixed with the background color to improve readability.
+Opaque images can also be converted to translucent ones with a few steps.
+
+A common usage is an effect similiar to previous qtermwidget versions or other terminal emulators.
+To achieve that, you can convert the background image to a translucent one with the transparency level matching the original terminal transparency.
+For example, if the original terminal transparency of qtermwidget was 25% (or 75% in some other terminal emulators), a converted image with transparency 25% will work as usual.
 The conversion can be done via ImageMagick, GraphicsMagick, GIMP or Krita.
 Here is an example command using ImageMagick:
 
     $ convert original_image.jpg -matte -channel A +level 0,25% +channel translucent_image.png
 
-The converted image can be specified as the terminal background image to achieve a similar effect to previous versions.
-Note that you also need to change the terminal transparency to 0% if you do not want to see another window or the desktop below the terminal.
+You may also want to change the terminal transparency to 0% if you do not want to see another window or the desktop below the terminal.

--- a/docs/configuration.md
+++ b/docs/configuration.md
@@ -1,0 +1,15 @@
+# Migration for background images
+
+How the background image is drawn has been changed since version 0.17.
+Intuitively, the background image is now drawn above the background color instead of below it.
+Technically, the background image is no longer blended with the background color.
+
+Before migrating the settings, the background image should be converted to a translucent one with the transparency level matching the original terminal transparency.
+For example, if the original terminal transparency of qtermwidget was 25%, the converted image should have transparency 25%.
+The conversion can be done via ImageMagick, GraphicsMagick, GIMP or Krita.
+Here is an example command using ImageMagick:
+
+    $ convert original_image.jpg -matte -channel A +level 0,25% +channel translucent_image.png
+
+The converted image can be specified as the terminal background image to achieve a similar effect to previous versions.
+Note that you also need to change the terminal transparency to 0% if you do not want to see another window or the desktop below the terminal.

